src/views/Combine.vue

```diff
@@ -5,6 +5,7 @@
         :projection="projection"
         :clusterMarkers="true"
         :layers="selectedLayers"
+        :getCustomSearchResults="getCustomSearchResults"
```

Comment:
Nice! The way it's implemented in this component in the looks looks pretty clean.